### ISSUE TITLE
[sdl2gdx 1.0.4] Failed loading libusb-1.0.dll

### ISSUE THREAD

**Issue by alyrow** [electronstudio/sdl2gdx#22]
Hello,
Well the flashing window is back, but I know how to fix that now.
But I have observed that in the traceback :
```traceback
EXTRACTED LOC C:\Users\____\AppData\Local\Temp\5687545216180750193.tmp
Joystick 0: XINPUT:Xbox 360 Controller
joystick org.libsdl.SDL_Joystick@917bcaf controller org.libsdl.SDL_GameController@916b7df
connected SDL GameController ShanWan PS3/PC Wired GamePad instance:0  guid: 030000005e0400008e02000000007801 v 0
NATIVE METHOD: loading from  "C:\Users\____\AppData\Local\Temp\5687545216180750193.tmp"
             result 64  Failed loading libusb-1.0.dll: The specified module could not be found
```

Is it normal ?

Best regards.

**Comment by electronstudio** [comment]
It seems to be something that was changed in SDL 2.0.12.  They went from disabling libusb on windows to enabling it but loading it dynamically.  So I would guess things will still work without it, but perhaps work better if you supply it.  If that's not the case then I could attempt to disable it completely.

**Comment by alyrow** [comment]
Yes the controller works, no crash but this occurs only when I close the game. Put  libusb-1.0.dll only makes it disappear.

(Off topic : the flashing window is back... Even the zip version doesn't fix...)

**Comment by electronstudio** [comment]
In that case I don't think it's a problem.  Possibly I should add libusb to the distribution if SDL if using it to improve controller detection.

If you can send me a version of the project with the flashing window I will take a look at it.

**Comment by alyrow** [comment]
Thanks : this is the [zip](https://drive.google.com/file/d/15qC4BZey3fRY-UiytdyP5-KNP8EU98J0/view?usp=sharing).
The flash occurs randomly.
Sorry, texts are not in english but it is not problematic.
If you want to use the controller to browse the cards, you need to press XBox/PS button then A/X then use buttons indiqued on cards. (Keyboard doesn't work for the moment)

**Comment by electronstudio** [comment]
Here is how it looks for me: https://www.youtube.com/watch?v=qkpCOc290-I&feature=youtu.be

It doesn't flash.  Can you send me a video of it flashing?

**Comment by alyrow** [comment]
Strange, it blinks for me : https://youtu.be/jfLtk3zjRlk

But with Github action, I have made a jar and this jar version doesn't flash.

**Comment by morvael** [comment]
Even placing libusb-1.0.dll in java.library.path doesn't make the message go away.

**Comment by isXander** [comment]
Cannot get joystick rumble working. Only difference between running directly in C (which works) vs in java is this error.

**Comment by electronstudio** [comment]
What OS and what controller are you using?

**Comment by isXander** [comment]
Fiddling with SDL hints fixed my issue. The SDL test had a window open which was probably the problem as I hadn't set SDL background hint (I was not using controller manager)

**Comment by Juancb28** [comment]
result 550  Failed loading libusb-1.0.dll: The specified module could not be found.

I'm using W11 with x64 and I did all the steps that were told in this page and I can't fix it
